app/modules/wallet/users/users.controller.ts

```diff
@@ -0,0 +1,54 @@
+import * as restify from 'restify';
+import { InternalServerError, NotFoundError } from 'restify-errors';
+import IController from '../../../interfaces/utils/IController';
+import { IRequest, IResponse } from '../../../interfaces/utils/IServer';
+import WalletUser from '../../../models/wallet_user.model';
+import { IWalletUser } from '../../../interfaces/models';
+
+export default class WalletUsersController implements IController {
+  public async post(req: IRequest, res: IResponse, next: restify.Next) {
+    try {
+      const walletUser = <IWalletUser> await WalletUser.create({
+        user_id: req.body.user_id,
+        client_id: req.client_id,
+        wallet_id: req.params.wallet
+      });
+
+      return res.send(walletUser);
+    } catch (e) {
+      console.log(e);
+      req.log.error(e);
+      throw new InternalServerError(e);
+    }
+    return next();
+  }
+
+  public async getAll(req: IRequest, res: IResponse, next: restify.Next) {
+    try {
+      const walletUsers = <IWalletUser[]> await WalletUser.findAll(
+        { where: { wallet_id: req.params.wallet } }
+      );
+      if (!walletUsers) {
+        return res.send(new NotFoundError('Users not found for given wallet id'));
+      }
+      return res.send(walletUsers);
+    } catch (e) {
+      req.log.error(e);
+      throw new InternalServerError(e);
+    }
+    return next();
+  }
+
+  public async delete(req: IRequest, res: IResponse, next: restify.Next) {
+    try {
+      const walletUser = await WalletUser.destroy({
+        where: { wallet_id: req.params.wallet, id: req.params.user }
+      });
+      return res.send();
```

Comment:
Check the responses guide here https://github.com/thinblock/tb-core-api/wiki/Contribution-Guidelines#responses